deeppavlov/skills/aiml_skill/aiml_skill.py

```diff
@@ -88,7 +88,7 @@ def _generate_user_id(self) -> str:
         """
         return uuid.uuid1().hex
 
-    def __call__(self, utterances_batch: List, history_batch: List, states_batch: List) -> Tuple[List, ...]:
+    def __call__(self, utterances_batch: List, history_batch: List=(), states_batch: List=()) -> Tuple[List, ...]:
```

Comment:
`utterances_batch: List[str]` would look better, since it's said in docstring that it should be a list of strings.
Also with typing like this mentioning "str" type can be skipped in the docstring.

Reply:
Fixed